### DISCUSSION TITLE
AML is Reloading User Config Files at Game Startup

### DISCUSSION THREAD

**Discussion by luxmanifestus** [X2CommunityCore/xcom2-launcher#398] [Help]
AML Version 1.5.1

I'm finding that AML is reloading the default game config files into the user (C:\Users\[USER]\Documents\My Games\XCOM2 War of the Chosen\XComGame\Config) config folder, overwriting any edits, without any active argument added to the AML settings, upon game startup.   

[settings.json](https://github.com/user-attachments/files/17048023/settings.json)
[AML.log](https://github.com/user-attachments/files/17048020/AML.log)

**Comment by RevZero** [comment]
Did it work before? if yes, what did you change? Maybe the XCOM game logs indicate what is going on? I am converting this to a discussion...

**Comment by Iridar** [comment]
This is what would happen if `-regenerateinis` argument was active. Can you make a screenshot of your AML settings window?

**Comment by luxmanifestus** [comment]
Yes, previous versions of AML would not overwrite the .inis in the user config directory without the -regenrateinis argument added in settings. Unofrtunately, Print Screen doesn't work while AML is active (another bug, lol?) so I can't show the settings window but '-regenerateinis' is not included in the Active Arguments field.

**Comment by RevZero** [comment]
Looking at the settings file you attached, the active arguments are 
```
  "ArgumentList": [
    "-review",
    "-noRedScreens",
    "-noStartUpMovies",
    "-allowConsole",
    "-force-d3d11"
  ],
```

I am not familiar with "-force-d3d11". Is this valid? Maybe try without?

**Comment by luxmanifestus** [comment]
The '-force-d3d11' argument makes the game run with DirectX 11. I tested load up with the argument removed and experienced the same behavior.
